CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
 - Add: NGSI-LD Unsupported Endpoints return valid error responses (#1277)
 - Fix: protect explicitAttrs evaluation of a bad formed expression (#1175)
 - Fix: ensure GET /ngsi-ld/v1/entities/<entity-id> returns a valid response (#1276)
+- Add: UT test case to check device and group based on name and type entity (#1279)
```

Comment:
```suggestion
```

Changelog is about bugfixing or functional changes. New tests of course are a good adition, but they are a "hardening" work that is not included in the changelog.